starter/src/assets/script.js

```diff
@@ -14,60 +12,80 @@
    - orange.jpg by Mae Mu
    - strawberry.jpg by Allec Gomes
 */
+  
+
+/* Create an array named productsArr and add all product object literals to array */
+
+
+/* Declare an empty array named cartArr to hold the items in the cart */
+
 
-/* Declare an empty array named cart to hold the items in the cart */
+// Helper function to get product
```

Comment:
I'm not sure what you mean here.  Please clarify.

Reply:
I was thinking there is no where in the curriculum where the concept of a helper function is explained and ultimately it is not necessary to write each function.  I think it is a place where some students could get blocked and discouraged very early in attempting the project.  Perhaps we could have the concept of refactoring parts of the core functions into a helper function in the stand out suggestions?

Reply:
That works